src/EditorComponent.jsx

```diff
@@ -163,6 +202,53 @@ const EditorComponent = props => {
             return true;
         };
 
+        // Editor command that simply tells the extension to capture the current frame. Does not apply any
+        // transaction to the state.
+        const tellExtensionToCaptureFrame = (state, dispatch) => {
```

Comment:
Parameters are no used.

Reply:
prosemirror-commands expects that the function take those two parameters. Thought it makes it easier to see that this is indeed a 'command'. Could either remove the two parameters or prepend _ to their name...